dmscripts/notify_buyers_to_award_closed_briefs.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime, date, timedelta
+
+import dmapiclient
+from dmutils.email.dm_notify import DMNotifyClient
+from dmutils.email.exceptions import EmailError
+from dmutils.formats import DATE_FORMAT
+
+from dmscripts.helpers import logging_helpers, date_helpers, brief_data_helpers
+from dmscripts.helpers.logging_helpers import logging
```

Comment:
do you need to import this if you've already imported `logging_helpers`? I think you only use it on the next line so could just do `logger = logging_helpers.configure_logger({'dmapiclient': logging_helpers.logging.INFO})`

Reply:
Ugh, this was also a c&p from somewhere else - I thought Pea and I had refactored this out but I guess it got missed. Will fix